### PR TITLE
Add missing migration

### PR DESCRIPTION
**Pull request by rfleschenberg**
I think we should include this migration (for the gallery plugin), but I am not entirely sure :)

For some reason, `manage.py runserver` does not complain about the fact that this migration is missing (does anyone know why?), but `manage.py makemigrations` creates it.

### CODE REVIEW

**Comment by jrief** [comment]
That's 'only' a proxy model and not applying it doesn't harm. I've never understood why it's important to migrate proxy models anyway.

**Comment by rfleschenberg** [comment]
Me neither. On the other hand, the fact that Django does it probably means that it *is* important :) Maybe the migration machinery needs it to get the dependency graph right or something? I tend to think it is safer to listen to Django here :)

**Comment by rfleschenberg** [comment]
If the user runs `manage.py makemigrations` in a project that uses cascade, Django will write a migration file like this into the cascade directory inside `site-packages`. I think we should avoid this. It is at best confusing. I believe that in principle, `site-packages` can even be read-only.

The user can work around this by specifying the apps to create migrations for (`manage.py makemigrations myapp`), but I think we should not expect people to do this.

It seems that the main reason Django creates these migration files is so ensure that the model options at this point in time are available for migrations (e.g. for `RunPython` operations), even if they are changed later -- thanks to @knbk for his input on this.

To sum up, I think we should merge this :)

**Comment by rfleschenberg** [comment]
@vnorbix made a very good point in issue #116: not adding the migration will likely break things for users if we roll out a "real" 0004 migration. I think that at the moment, the safest thing to do therefore is to add this migration, so I am merging this PR. Maybe we can figure out a better way later.
